### DISCUSSION TITLE
Add blips using request

### DISCUSSION THREAD

**Discussion by maikelmo** [TGRHavoc/live_map-interface#57] [General]
Hi!

Is there a way to insert blips into blips.json using a web request?

I want to insert blips which are defined in a control panel.

**Comment by TGRHavoc** [comment]
No, but you can add then manually. You can either go to each place and and do the command in game or, edit the json file manually. I don't plan on adding anything to the resource to allow you to add blips using a request. The resource isn't secured so, someone could just spam create blips and potentially crash the server.

Could you elaborate on what exactly you mean? It's just that you've asked this on the interface github and I feel like I may have missed what you're actually trying to accomplish.

**Comment by maikelmo** [comment]
Thanks for your quick response.

I'm trying to create a web portal where users can create own blips based on postal codes. I don't really know which postal codes they want to use. That's why I want to create a web portal to interact with the blips.json

**Comment by TGRHavoc** [comment]
@maikelmo you could implement this fully on the client's side. You could implement it [on click on the map object](https://leafletjs.com/reference-1.7.1.html#event-objects). And store them in localstorage. Then, check if the markers exist on local storage next time the page loads. This would give each user their own, unique blips that they can create themselves.

**Comment by maikelmo** [comment]
Thanks! Do you have an example for blips.json?
